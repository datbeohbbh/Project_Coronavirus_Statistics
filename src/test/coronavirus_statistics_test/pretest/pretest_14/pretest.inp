20 27
q 1 7 47
q 1 6 104
? 4 2
q 1 20 96
? 4 3
? 4 1
? 4 3
q 2 6 55
q 2 6 47
q 2 20 84
? 4 2
q 3 6 2
q 3 20 1
? 4 1
? 4 2
? 4 1
? 4 2
q 1 1 77
? 4 3
? 6
? 7
? 4 3
? 4 3
? 7
? 4 1
? 7
? 7