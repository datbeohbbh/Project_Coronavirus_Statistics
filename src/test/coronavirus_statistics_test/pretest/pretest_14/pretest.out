Query ? 4 2 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 104
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 104
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 47
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 104
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 12
-> Has recovered : 84
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 11
-> Has recovered : 84
-> Was dead : 1

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 47
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 11
-> Has recovered : 84
-> Was dead : 1

Query ? 4 2 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 47
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 77
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 20
Quantity: 84 people

Query ? 7 :
The city with the highest death is city with the index 20
Quantity: 1 people

Query ? 4 3 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 77
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 77
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 20
Quantity: 1 people

Query ? 4 1 :
The answer is the city with the index 20 
- Information of the city 20 -
-> Number of current patients : 11
-> Has recovered : 84
-> Was dead : 1

Query ? 7 :
The city with the highest death is city with the index 20
Quantity: 1 people

Query ? 7 :
The city with the highest death is city with the index 20
Quantity: 1 people

